contrib/cluster-backup/README.md

```diff
@@ -0,0 +1,37 @@
+# Restore
+Restoring a Kubernetes environment (from a previous backup) may be executed through the use of the bash script: [kubernetes-on-aws-restore.sh](/Documentation/scripts/kubernetes-on-aws-backup.sh)    
```

Comment:
I think this link should be to `contrib/cluster-backup/restore.sh` rather than the docs backup.sh script?